src/paperbot/application/services/resend_service.py

```
@@ -0,0 +1,99 @@
+
+        for email_addr in to:
+            token = unsub_tokens.get(email_addr, "")
+            unsub_link = f"{self.unsub_base_url}/api/newsletter/unsubscribe/{token}"
+            html_body = self._render_html(report, markdown, unsub_link)
+            text = self._render_text(report, markdown, unsub_link)
+            try:
+                r = self.send(
+                    to=[email_addr], subject=subject, html_body=html_body, text=text
+                )
+                results[email_addr] = {"ok": True, "id": r.get("id")}
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)